Pomodoro/app/src/main/java/no/uia/ikt205/pomodoro/MainActivity.kt

```diff
@@ -2,89 +2,152 @@ package no.uia.ikt205.pomodoro
 
 import android.os.Bundle
 import android.os.CountDownTimer
-import android.view.Gravity
 import android.view.View
 import android.widget.*
 import androidx.appcompat.app.AppCompatActivity
 import no.uia.ikt205.pomodoro.util.millisecondsToDescriptiveTime
+import java.nio.file.Files.size
 
 
-class MainActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener {
+class MainActivity : AppCompatActivity() {
 
     lateinit var timer:CountDownTimer
+    lateinit var pauseTimer:CountDownTimer
     lateinit var startButton:Button
     lateinit var coutdownDisplay:TextView
-    lateinit var dropDown:Spinner
+    lateinit var timeSelectSlider:SeekBar
+    lateinit var repetitionInput:EditText
+    lateinit var isPauseText:TextView
+    lateinit var pauseTimeSelect:SeekBar
 
     private var hasTimerStarted: Boolean = false
+    private var hasPauseTimerStarted: Boolean = false
 
-    private val countdownTimeArray = arrayOf<Long>(
-        1800000L,
-        3600000L,
-        5400000L,
-        7200000L
-    )
-    private var timeToCountDownInMs = countdownTimeArray[0]
+    // Default timer value is 15 minutes
+    private var timeToCountDownInMs = 15 * 60000L // Default 15 minutes
     private val timeTicks = 1000L
+    private var pauseTime = 15 * 60000L // Default 15 minutes
+
+    // Default amount of repetitions
+    private var timerRepetitionAmount:Int = 0
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        dropDown = findViewById<View>(R.id.dropdown_menuTimes) as Spinner
-        val aa = ArrayAdapter(this, android.R.layout.simple_spinner_item, countdownTimeArray)
-        aa.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+        isPauseText = findViewById(R.id.isPauseText)
+        isPauseText.visibility = View.INVISIBLE
+
+        timeSelectSlider = findViewById<SeekBar>(R.id.timeSelectSlider)
+        timeSelectSlider.setOnSeekBarChangeListener(object: SeekBar.OnSeekBarChangeListener{
+            override fun onProgressChanged(seekBar: SeekBar?, progress: Int, fromUser: Boolean) {
+                if (hasTimerStarted) {
+                    timer.cancel()
+                    hasTimerStarted = false
+                }
+                timeToCountDownInMs = progress * 60000L
```

Comment:
Her kunne du også gjerne beskrevet hva 60000L er for noe eventuelt brukt en funksjon for utregningen slik at koden blir ryddigere :)